src/main/java/de/muenchen/isi/infrastructure/entity/stammdaten/Umlegung.java

```diff
@@ -0,0 +1,33 @@
+package de.muenchen.isi.infrastructure.entity.stammdaten;
+
+import de.muenchen.isi.infrastructure.entity.BaseEntity;
+import java.math.BigDecimal;
+import java.time.LocalDate;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+@Entity
+@Data
+@EqualsAndHashCode
+public class Umlegung extends BaseEntity {
```

Comment:
Die Umrechnung der Förderarten ist durch diese Stammdatenentität auf zwei Zielförderarten begrenzt.

Das Stammdatum ist generisch zu implementieren. Beispiel:

Die Zielumlage (hier UmlageFoerderart) kann in der Stammdatenentität sogar als JSON-Attribut hinterlegt werden, da es sich im Sinne von DDD um ein Value-Object handelt. Somit wird die UmlageFoerderart keine eigene Tabelle erstellt.

```java
public class UmlegungFoerderart extends BaseEntity {

    @Column(nullable = false)
    private String bezeichnung;

    @Column(nullable = false)
    private LocalDate gueltigAb;

    // Die Klasse Foerderart existiert bereits.
    @Type(type = "json")
    @Column(columnDefinition = "jsonb")
    private Set<Foerderart> umlegungsschluessel;

}



```